src/app/_constants/paths.ts

```diff
@@ -73,6 +75,7 @@ export const PATHS = {
     '/security/coordinated-disclosure-policy',
     'Coordinated Disclosure Policy',
   ),
+  BUG_BOUNTY: createPathObject('/security/bug-bounty', 'Bug Bounty Program'),
```

Comment:
At some point we could also refactor this, similar to what you did with `testPageMetadata`? passing a path: "", and a label: ""